lesson_03/average/average.go

```diff
@@ -0,0 +1,92 @@
+package average
+
+import (
+	"fmt"
+	"reflect"
+)
+
+type ReturnCode int8
+
+const (
+	Success ReturnCode = iota
+	Error
+)
+
+func isNumericKind(kind reflect.Kind) bool {
```

Comment:
Maybe there is a standard way to do the same? I did not find.

Reply:
I guess that here is two ways `ParseFloat` or the way you do. I don't know any such things like `std::is_integral`=)